Specs/DateTime/Spanish/DatePeriodExtractor.json

```diff
@@ -538,28 +412,6 @@
       }
     ]
   },
-  {
-    "Input": "Estare afuera la tercer semana de 2027",
```

Comment:
We could change this test to `Estare afuera la tercer semana del 2027` instead of deleting it.

Reply:
Thanks!